conda/recipes/librmm/meta.yaml

```diff
@@ -20,7 +20,14 @@ requirements:
     - {{ compiler('cuda') }} {{ cuda_version }}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cudatoolkit {{ cuda_version }}.*
+    - cudatoolkit ={{ cuda_version }}
+    # We require spdlog and fmt (which was devendored from spdlog
+    # conda-forge packages in 1.11.0) so that the spdlog headers are not
+    # pulled by CPM and installed as a part of the rmm packages. However,
+    # building against librmm still requires these headers. They are also
+    # added as a run requirement via the packages' run_exports.
```

Comment:
@kkraus14 Will these be added as a run requirement of both or neither of `librmm` and `librmm-tests` because of the `run_exports`? Should I manually re-specify them in `librmm`'s `run` section?

Reply:
I'm building the package locally now to check. I'm not quite sure the rules on output packages as it's kinda entirely undocumented.

Reply:
From looking at other recipes, I think we do need to specify them again as `librmm` run dependencies. Added in b7e8154b215b2ed08fcf4ce46d7481eff6c68920.

Reply:
> From looking at other recipes, I think we do need to specify them again as `librmm` run dependencies. Added in [b7e8154](https://github.com/rapidsai/rmm/commit/b7e8154b215b2ed08fcf4ce46d7481eff6c68920).

I can confirm this.

`run_exports` from the top-level package aren't propagated to `output` packages.

We noticed this behavior when we first consolidated the recipes back in the day.